src/cli/commands/force/lightning/local/device/__tests__/create.test.ts

```diff
@@ -4,13 +4,16 @@
  * SPDX-License-Identifier: MIT
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
+
+// tslint:disable: no-submodule-imports
```

Comment:
Do we know if this is going to go away when we publish the package? I'd like to presume it's because it wasn't installed from a registry?  Anyway, if it's *not* going away, we probably want to whitelist the necessities in `tslint.json`, rather than clutter the code with the disable statements.

Reply:
@khawkins If I were to use `import core from '@salesforce/lwc-dev-mobile-core'` then the compiler throws this error:
```
Cannot find module '@salesforce/lwc-dev-mobile-core' or its corresponding type declarations.
```

I'm not sure if this is going to go away when we publish the package. Perhaps it's best to just whitelist in `tslint.json` ?

Reply:
Discussed this with @maliroteh-sf offline, and we're just going to disable this rule in our project config, until we figure out if there's a better way to reference submodule code paths in import statements.

Reply:
LDS(now Luvio) has a top level `main.ts` that consolidates `export`-able in a project that are in different different paths.

https://github.com/salesforce/luvio/blob/master/packages/%40luvio/compiler/src/main.ts

Reply:
That's an interesting approach, although that's yet another list that must be maintained.